app/index.js

```diff
@@ -10,6 +10,23 @@ function checkWater() {
   console.log(orange('Check your pets water and food, make sure its all fine :)'));
 }
 
+var cat_ascii = `  	
```

Comment:
Change this var to `const`.  We have a .eslintrc.json file in this repo to check for this in your editor/test when you commit.  It won't pass travis-ci which is why your PR is labeled with a red X